index.js

```diff
@@ -151,10 +151,10 @@ module.exports.processV2 = (output, origin) => {
 };
 
 module.exports.processPlaylist = (output) => {
-  return output.toString().trim().split('\n').map((string) => {
-    const video = JSON.parse(string);
-    return `https://www.youtube.com/watch?v=${video.id}`;
-  });
+  const outputJSON = JSON.parse(output)
+  if (outputJSON["entries"]) {
+    return outputJSON["entries"].map((video) => `https://www.youtube.com/watch?v=${video.id}`);
+  }
 };
```

Comment:
I think this should return an empty array if `outputJSON["entries"]` is not set.

Reply:
Yeah that would be more consistent with the old implementation. Fixed.